packaging/nnstreamer.spec

```diff
@@ -40,6 +40,8 @@ BuildRequires: coregl-devel
 BuildRequires: cairo-devel
 # custom_example_opencv filter requires opencv-devel
 BuildRequires: opencv-devel
+# For './testAll.sh' time limit.
```

Comment:
It's good for maintenance even though base packages of 'gbs build' already includes this package.